doc/source/ray-core/scheduling/resources.rst

```diff
@@ -176,3 +182,7 @@ The precision of the fractional resource requirement is 0.0001 so you should avo
 
   Besides resource requirements, you can also specify an environment for a task or actor to run in,
   which can include Python packages, local files, environment variables, and more---see :ref:`Runtime Environments <runtime-environments>` for details.
+
+Dynamic Resource Update
+-----------------------
+Ray **does not allow to dynamically update of resources capacities after a node starts**.
```

Comment:
Nit:

"Ray does not allow resource capacities to be dynamically updated ..."

Reply:
Ha ChatGPT says:

"Ray does not permit dynamic updates of resource capacities after a node has begun." 😄

Reply:
should I follow ChatGPT suggestion? lol

Reply:
addressed (as a chatgpt version)